GUI/src/main/kotlin/interface/AbstractFrameNavigationInterface.kt

```diff
@@ -0,0 +1,30 @@
+
+/**
+ * Provides navigation methods for frames in a video.
+ *
+ * This interface defines methods to navigate frames in various ways, including jumping a specified number
+ * of frames in a given direction, jumping to a specific percentage of the video, jumping to a specific frame,
+ * and jumping to the next difference frame. Implementing classes should provide implementation for these methods.
+ */
+interface AbstractFrameNavigationInterface {
+
+    /**
+     * Jump n frames in a specified direction
+     */
+    abstract fun jumpFrames(frames: Int, forward: Boolean)
```

Comment:
The abstract does not have a purpose here, as this is now an interface and by definition (implicitly) all functions are abstract anyways :)